fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/SortPhase.java

```diff
@@ -0,0 +1,45 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans;
+
+/**
+ * Represents different phase of agg and map it to the
+ * enum of agg phase definition of stale optimizer.
+ */
+public enum SortPhase {
+    MERGE_SORT("MergeSort"),
+    GATHER_SORT("GatherSort"),
```

Comment:
consider following plan
```
sort(gather)
  +---scan(T)
```
if T is replica table, do we add an exchange to gather data?

Reply:
Do you mean T has only one tablet? 
In this case, the scan node has Gather property. So the gather is satisfied by the child of `sort`. We don't need to add a gather exchange node